lib/spack/spack/schema/config.py

```diff
@@ -90,7 +90,13 @@
             'additional_external_search_paths': {
                 'type': 'array',
                 'items': {'type': 'string'}
-            }
+            },
+            'flags': {
+                'type': 'object',
+                'properties': {
+                    'werror': { 'type': 'string', 'enum': ['all','specific','none'], },
```

Comment:
Wow @trws that was fast! Just to check my understanding, under this PR the default config for Spack (`none`) will remove all Werrors from builds? Or would that be achieved by setting `all`?

Reply:
Yes, it keeps none of them.